ztp/kube-compare-reference/metadata.yaml

```diff
@@ -204,5 +204,7 @@ fieldsToOmit:
       - pathToKey: metadata.labels."pod-security.kubernetes.io/warn-version"
       - pathToKey: metadata.annotations."capability.openshift.io/name"
       - pathToKey: metadata.annotations."olm.providedAPIs"
+      - pathToKey: metadata.annotations."operator.sriovnetwork.openshift.io/last-network-namespace"
```

Comment:
Verified this is an operator managed field used in tracking where NAD are created for SriovNetworks. 
https://github.com/openshift/sriov-network-operator/blob/master/controllers/generic_network_controller.go#L141-L152